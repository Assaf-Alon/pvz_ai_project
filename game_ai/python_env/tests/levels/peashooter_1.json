{
    "50": [["Normal", 0]]
}
