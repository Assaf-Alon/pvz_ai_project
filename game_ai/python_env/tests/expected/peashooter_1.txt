[100] Sun autospawned. Total: 75.
[200] Sun autospawned. Total: 100.
[201] Planted Peashooter in (0, 0). Total: 0.
[216] Pea generated by Peashooter in (0, 0).
[216] Pea in (0, 0) moved to (0, 1).
[218] Pea in (0, 1) moved to (0, 2).
[220] Pea in (0, 2) moved to (0, 3).
[222] Pea in (0, 3) moved to (0, 4).
[224] Pea in (0, 4) moved to (0, 5).
[226] Pea in (0, 5) moved to (0, 6).
[228] Pea in (0, 6) moved to (0, 7).
[230] Pea in (0, 7) moved to (0, 8).
[231] Pea generated by Peashooter in (0, 0).
[231] Pea in (0, 0) moved to (0, 1).
[232] Pea in (0, 8) moved to (0, 9).
[233] Pea in (0, 1) moved to (0, 2).
[234] Pea in (0, 9) moved to (0, 10).
[234] Pea in (0, 10) removed.
[235] Pea in (0, 2) moved to (0, 3).
[237] Pea in (0, 3) moved to (0, 4).
[239] Pea in (0, 4) moved to (0, 5).
[241] Pea in (0, 5) moved to (0, 6).
[243] Pea in (0, 6) moved to (0, 7).
[245] Pea in (0, 7) moved to (0, 8).
[246] Pea generated by Peashooter in (0, 0).
[246] Pea in (0, 0) moved to (0, 1).
[247] Pea in (0, 8) moved to (0, 9).
[248] Pea in (0, 1) moved to (0, 2).
[249] Pea in (0, 9) moved to (0, 10).
[249] Pea in (0, 10) removed.
[250] Pea in (0, 2) moved to (0, 3).
[252] Pea in (0, 3) moved to (0, 4).
[254] Pea in (0, 4) moved to (0, 5).
[256] Pea in (0, 5) moved to (0, 6).
[258] Pea in (0, 6) moved to (0, 7).
[260] Pea in (0, 7) moved to (0, 8).
[261] Pea generated by Peashooter in (0, 0).
[261] Pea in (0, 0) moved to (0, 1).
[262] Pea in (0, 8) moved to (0, 9).
[263] Pea in (0, 1) moved to (0, 2).
[264] Pea in (0, 9) moved to (0, 10).
[264] Pea in (0, 10) removed.
[265] Pea in (0, 2) moved to (0, 3).
[267] Pea in (0, 3) moved to (0, 4).
[269] Pea in (0, 4) moved to (0, 5).
[271] Pea in (0, 5) moved to (0, 6).
[273] Pea in (0, 6) moved to (0, 7).
[275] Pea in (0, 7) moved to (0, 8).
[276] Pea generated by Peashooter in (0, 0).
[276] Pea in (0, 0) moved to (0, 1).
[277] Pea in (0, 8) moved to (0, 9).
[278] Pea in (0, 1) moved to (0, 2).
[279] Pea in (0, 9) moved to (0, 10).
[279] Pea in (0, 10) removed.
[280] Pea in (0, 2) moved to (0, 3).
[282] Pea in (0, 3) moved to (0, 4).
[284] Pea in (0, 4) moved to (0, 5).
[286] Pea in (0, 5) moved to (0, 6).
[288] Pea in (0, 6) moved to (0, 7).
[290] Pea in (0, 7) moved to (0, 8).
[291] Pea generated by Peashooter in (0, 0).
[291] Pea in (0, 0) moved to (0, 1).
[292] Pea in (0, 8) moved to (0, 9).
[293] Pea in (0, 1) moved to (0, 2).
[294] Pea in (0, 9) moved to (0, 10).
[294] Pea in (0, 10) removed.
[295] Pea in (0, 2) moved to (0, 3).
[297] Pea in (0, 3) moved to (0, 4).
[299] Pea in (0, 4) moved to (0, 5).
[300] Sun autospawned. Total: 25.
[301] Pea in (0, 5) moved to (0, 6).
[303] Pea in (0, 6) moved to (0, 7).
[305] Pea in (0, 7) moved to (0, 8).
[306] Pea generated by Peashooter in (0, 0).
[306] Pea in (0, 0) moved to (0, 1).
[307] Pea in (0, 8) moved to (0, 9).
[308] Pea in (0, 1) moved to (0, 2).
[309] Pea in (0, 9) moved to (0, 10).
[309] Pea in (0, 10) removed.
[310] Pea in (0, 2) moved to (0, 3).
[312] Pea in (0, 3) moved to (0, 4).
[314] Pea in (0, 4) moved to (0, 5).
[316] Pea in (0, 5) moved to (0, 6).
[318] Pea in (0, 6) moved to (0, 7).
[320] Pea in (0, 7) moved to (0, 8).
[321] Pea generated by Peashooter in (0, 0).
[321] Pea in (0, 0) moved to (0, 1).
[322] Pea in (0, 8) moved to (0, 9).
[323] Pea in (0, 1) moved to (0, 2).
[324] Pea in (0, 9) moved to (0, 10).
[324] Pea in (0, 10) removed.
[325] Pea in (0, 2) moved to (0, 3).
[327] Pea in (0, 3) moved to (0, 4).
[329] Pea in (0, 4) moved to (0, 5).
[331] Pea in (0, 5) moved to (0, 6).
[333] Pea in (0, 6) moved to (0, 7).
[335] Pea in (0, 7) moved to (0, 8).
[336] Pea generated by Peashooter in (0, 0).
[336] Pea in (0, 0) moved to (0, 1).
[337] Pea in (0, 8) moved to (0, 9).
[338] Pea in (0, 1) moved to (0, 2).
[339] Pea in (0, 9) moved to (0, 10).
[339] Pea in (0, 10) removed.
[340] Pea in (0, 2) moved to (0, 3).
[342] Pea in (0, 3) moved to (0, 4).
[344] Pea in (0, 4) moved to (0, 5).
[346] Pea in (0, 5) moved to (0, 6).
[348] Pea in (0, 6) moved to (0, 7).
[350] Pea in (0, 7) moved to (0, 8).
[351] Pea generated by Peashooter in (0, 0).
[351] Pea in (0, 0) moved to (0, 1).
[352] Pea in (0, 8) moved to (0, 9).
[353] Pea in (0, 1) moved to (0, 2).
[354] Pea in (0, 9) moved to (0, 10).
[354] Pea in (0, 10) removed.
[355] Pea in (0, 2) moved to (0, 3).
[357] Pea in (0, 3) moved to (0, 4).
[359] Pea in (0, 4) moved to (0, 5).
[361] Pea in (0, 5) moved to (0, 6).
[363] Pea in (0, 6) moved to (0, 7).
[365] Pea in (0, 7) moved to (0, 8).
[366] Pea generated by Peashooter in (0, 0).
[366] Pea in (0, 0) moved to (0, 1).
[367] Pea in (0, 8) moved to (0, 9).
[368] Pea in (0, 1) moved to (0, 2).
[369] Pea in (0, 9) moved to (0, 10).
[369] Pea in (0, 10) removed.
[370] Pea in (0, 2) moved to (0, 3).
[372] Pea in (0, 3) moved to (0, 4).
[374] Pea in (0, 4) moved to (0, 5).
[376] Pea in (0, 5) moved to (0, 6).
[378] Pea in (0, 6) moved to (0, 7).
[380] Pea in (0, 7) moved to (0, 8).
[381] Pea generated by Peashooter in (0, 0).
[381] Pea in (0, 0) moved to (0, 1).
[382] Pea in (0, 8) moved to (0, 9).
[383] Pea in (0, 1) moved to (0, 2).
[384] Pea in (0, 9) moved to (0, 10).
[384] Pea in (0, 10) removed.
[385] Pea in (0, 2) moved to (0, 3).
[387] Pea in (0, 3) moved to (0, 4).
[389] Pea in (0, 4) moved to (0, 5).
[391] Pea in (0, 5) moved to (0, 6).
[393] Pea in (0, 6) moved to (0, 7).
[395] Pea in (0, 7) moved to (0, 8).
[396] Pea generated by Peashooter in (0, 0).
[396] Pea in (0, 0) moved to (0, 1).
[397] Pea in (0, 8) moved to (0, 9).
[398] Pea in (0, 1) moved to (0, 2).
[399] Pea in (0, 9) moved to (0, 10).
[399] Pea in (0, 10) removed.
[400] Pea in (0, 2) moved to (0, 3).
[400] Sun autospawned. Total: 50.
[402] Pea in (0, 3) moved to (0, 4).
[404] Pea in (0, 4) moved to (0, 5).
[406] Pea in (0, 5) moved to (0, 6).
[408] Pea in (0, 6) moved to (0, 7).
[410] Pea in (0, 7) moved to (0, 8).
[411] Pea generated by Peashooter in (0, 0).
[411] Pea in (0, 0) moved to (0, 1).
[412] Pea in (0, 8) moved to (0, 9).
[413] Pea in (0, 1) moved to (0, 2).
[414] Pea in (0, 9) moved to (0, 10).
[414] Pea in (0, 10) removed.
[415] Pea in (0, 2) moved to (0, 3).
[417] Pea in (0, 3) moved to (0, 4).
[419] Pea in (0, 4) moved to (0, 5).
[421] Pea in (0, 5) moved to (0, 6).
[423] Pea in (0, 6) moved to (0, 7).
[425] Pea in (0, 7) moved to (0, 8).
[426] Pea generated by Peashooter in (0, 0).
[426] Pea in (0, 0) moved to (0, 1).
[427] Pea in (0, 8) moved to (0, 9).
[428] Pea in (0, 1) moved to (0, 2).
[429] Pea in (0, 9) moved to (0, 10).
[429] Pea in (0, 10) removed.
[430] Pea in (0, 2) moved to (0, 3).
[432] Pea in (0, 3) moved to (0, 4).
[434] Pea in (0, 4) moved to (0, 5).
[436] Pea in (0, 5) moved to (0, 6).
[438] Pea in (0, 6) moved to (0, 7).
[440] Pea in (0, 7) moved to (0, 8).
[441] Pea generated by Peashooter in (0, 0).
[441] Pea in (0, 0) moved to (0, 1).
[442] Pea in (0, 8) moved to (0, 9).
[443] Pea in (0, 1) moved to (0, 2).
[444] Pea in (0, 9) moved to (0, 10).
[444] Pea in (0, 10) removed.
[445] Pea in (0, 2) moved to (0, 3).
[447] Pea in (0, 3) moved to (0, 4).
[449] Pea in (0, 4) moved to (0, 5).
[451] Pea in (0, 5) moved to (0, 6).
[453] Pea in (0, 6) moved to (0, 7).
[455] Pea in (0, 7) moved to (0, 8).
[456] Pea generated by Peashooter in (0, 0).
[456] Pea in (0, 0) moved to (0, 1).
[457] Pea in (0, 8) moved to (0, 9).
[458] Pea in (0, 1) moved to (0, 2).
[459] Pea in (0, 9) moved to (0, 10).
[459] Pea in (0, 10) removed.
[460] Pea in (0, 2) moved to (0, 3).
[462] Pea in (0, 3) moved to (0, 4).
[464] Pea in (0, 4) moved to (0, 5).
[466] Pea in (0, 5) moved to (0, 6).
[468] Pea in (0, 6) moved to (0, 7).
[470] Pea in (0, 7) moved to (0, 8).
[471] Pea generated by Peashooter in (0, 0).
[471] Pea in (0, 0) moved to (0, 1).
[472] Pea in (0, 8) moved to (0, 9).
[473] Pea in (0, 1) moved to (0, 2).
[474] Pea in (0, 9) moved to (0, 10).
[474] Pea in (0, 10) removed.
[475] Pea in (0, 2) moved to (0, 3).
[477] Pea in (0, 3) moved to (0, 4).
[479] Pea in (0, 4) moved to (0, 5).
[481] Pea in (0, 5) moved to (0, 6).
[483] Pea in (0, 6) moved to (0, 7).
[485] Pea in (0, 7) moved to (0, 8).
[486] Pea generated by Peashooter in (0, 0).
[486] Pea in (0, 0) moved to (0, 1).
[487] Pea in (0, 8) moved to (0, 9).
[488] Pea in (0, 1) moved to (0, 2).
[489] Pea in (0, 9) moved to (0, 10).
[489] Pea in (0, 10) removed.
[490] Pea in (0, 2) moved to (0, 3).
[492] Pea in (0, 3) moved to (0, 4).
[494] Pea in (0, 4) moved to (0, 5).
[496] Pea in (0, 5) moved to (0, 6).
[498] Pea in (0, 6) moved to (0, 7).
[500] Pea in (0, 7) moved to (0, 8).
[500] Normal zombie being spawned at lane 0
[500] Sun autospawned. Total: 75.
[501] Pea generated by Peashooter in (0, 0).
[501] Pea in (0, 0) moved to (0, 1).
[502] Pea in (0, 8) moved to (0, 9).
[502] Pea in (0, 9) attacked.
[502] Normal zombie in (0, 9) was damaged. HP: 161.
[502] Pea in (0, 9) removed.
[503] Pea in (0, 1) moved to (0, 2).
[505] Pea in (0, 2) moved to (0, 3).
[507] Pea in (0, 3) moved to (0, 4).
[509] Pea in (0, 4) moved to (0, 5).
[511] Pea in (0, 5) moved to (0, 6).
[513] Pea in (0, 6) moved to (0, 7).
[515] Pea in (0, 7) moved to (0, 8).
[516] Pea generated by Peashooter in (0, 0).
[516] Pea in (0, 0) moved to (0, 1).
[517] Pea in (0, 8) moved to (0, 9).
[517] Pea in (0, 9) attacked.
[517] Normal zombie in (0, 9) was damaged. HP: 141.
[517] Pea in (0, 9) removed.
[518] Pea in (0, 1) moved to (0, 2).
[520] Pea in (0, 2) moved to (0, 3).
[522] Pea in (0, 3) moved to (0, 4).
[524] Pea in (0, 4) moved to (0, 5).
[526] Pea in (0, 5) moved to (0, 6).
[528] Pea in (0, 6) moved to (0, 7).
[530] Pea in (0, 7) moved to (0, 8).
[531] Pea generated by Peashooter in (0, 0).
[531] Pea in (0, 0) moved to (0, 1).
[532] Pea in (0, 8) moved to (0, 9).
[532] Pea in (0, 9) attacked.
[532] Normal zombie in (0, 9) was damaged. HP: 121.
[532] Pea in (0, 9) removed.
[533] Pea in (0, 1) moved to (0, 2).
[535] Pea in (0, 2) moved to (0, 3).
[537] Pea in (0, 3) moved to (0, 4).
[539] Pea in (0, 4) moved to (0, 5).
[541] Pea in (0, 5) moved to (0, 6).
[543] Pea in (0, 6) moved to (0, 7).
[545] Pea in (0, 7) moved to (0, 8).
[546] Pea generated by Peashooter in (0, 0).
[546] Pea in (0, 0) moved to (0, 1).
[547] Pea in (0, 8) moved to (0, 9).
[547] Pea in (0, 9) attacked.
[547] Normal zombie in (0, 9) was damaged. HP: 101.
[547] Pea in (0, 9) removed.
[547] Normal zombie in (0, 9) moved to (0, 8).
[548] Pea in (0, 1) moved to (0, 2).
[550] Pea in (0, 2) moved to (0, 3).
[552] Pea in (0, 3) moved to (0, 4).
[554] Pea in (0, 4) moved to (0, 5).
[556] Pea in (0, 5) moved to (0, 6).
[558] Pea in (0, 6) moved to (0, 7).
[560] Pea in (0, 7) moved to (0, 8).
[560] Pea in (0, 8) attacked.
[560] Normal zombie in (0, 8) was damaged. HP: 81.
[560] Pea in (0, 8) removed.
[561] Pea generated by Peashooter in (0, 0).
[561] Pea in (0, 0) moved to (0, 1).
[563] Pea in (0, 1) moved to (0, 2).
[565] Pea in (0, 2) moved to (0, 3).
[567] Pea in (0, 3) moved to (0, 4).
[569] Pea in (0, 4) moved to (0, 5).
[571] Pea in (0, 5) moved to (0, 6).
[573] Pea in (0, 6) moved to (0, 7).
[575] Pea in (0, 7) moved to (0, 8).
[575] Pea in (0, 8) attacked.
[575] Normal zombie in (0, 8) was damaged. HP: 61.
[575] Pea in (0, 8) removed.
[576] Pea generated by Peashooter in (0, 0).
[576] Pea in (0, 0) moved to (0, 1).
[578] Pea in (0, 1) moved to (0, 2).
[580] Pea in (0, 2) moved to (0, 3).
[582] Pea in (0, 3) moved to (0, 4).
[584] Pea in (0, 4) moved to (0, 5).
[586] Pea in (0, 5) moved to (0, 6).
[588] Pea in (0, 6) moved to (0, 7).
[590] Pea in (0, 7) moved to (0, 8).
[590] Pea in (0, 8) attacked.
[590] Normal zombie in (0, 8) was damaged. HP: 41.
[590] Pea in (0, 8) removed.
[591] Pea generated by Peashooter in (0, 0).
[591] Pea in (0, 0) moved to (0, 1).
[593] Pea in (0, 1) moved to (0, 2).
[594] Normal zombie in (0, 8) moved to (0, 7).
[595] Pea in (0, 2) moved to (0, 3).
[597] Pea in (0, 3) moved to (0, 4).
[599] Pea in (0, 4) moved to (0, 5).
[600] Sun autospawned. Total: 100.
[601] Pea in (0, 5) moved to (0, 6).
[601] Planted Peashooter in (1, 0). Total: 0.
[603] Pea in (0, 6) moved to (0, 7).
[603] Pea in (0, 7) attacked.
[603] Normal zombie in (0, 7) was damaged. HP: 21.
[603] Pea in (0, 7) removed.
[606] Pea generated by Peashooter in (0, 0).
[606] Pea in (0, 0) moved to (0, 1).
[608] Pea in (0, 1) moved to (0, 2).
[610] Pea in (0, 2) moved to (0, 3).
[612] Pea in (0, 3) moved to (0, 4).
[614] Pea in (0, 4) moved to (0, 5).
[616] Pea generated by Peashooter in (1, 0).
[616] Pea in (0, 5) moved to (0, 6).
[616] Pea in (1, 0) moved to (1, 1).
[618] Pea in (0, 6) moved to (0, 7).
[618] Pea in (0, 7) attacked.
[618] Normal zombie in (0, 7) was damaged. HP: 1.
[618] Pea in (0, 7) removed.
[618] Pea in (1, 1) moved to (1, 2).
[620] Pea in (1, 2) moved to (1, 3).
[621] Pea generated by Peashooter in (0, 0).
[621] Pea in (0, 0) moved to (0, 1).
[622] Pea in (1, 3) moved to (1, 4).
[623] Pea in (0, 1) moved to (0, 2).
[624] Pea in (1, 4) moved to (1, 5).
[625] Pea in (0, 2) moved to (0, 3).
[626] Pea in (1, 5) moved to (1, 6).
[627] Pea in (0, 3) moved to (0, 4).
[628] Pea in (1, 6) moved to (1, 7).
[629] Pea in (0, 4) moved to (0, 5).
[630] Pea in (1, 7) moved to (1, 8).
[631] Pea generated by Peashooter in (1, 0).
[631] Pea in (0, 5) moved to (0, 6).
[631] Pea in (1, 0) moved to (1, 1).
[632] Pea in (1, 8) moved to (1, 9).
[633] Pea in (0, 6) moved to (0, 7).
[633] Pea in (0, 7) attacked.
[633] Normal zombie in (0, 7) was damaged. HP: -19.
[633] Normal zombie in (0, 7) was killed.
[633] Pea in (0, 7) removed.
[633] Pea in (1, 1) moved to (1, 2).
[633] You've won!
